crates/vision/src/robot_detection.rs

```diff
@@ -94,15 +98,23 @@ impl RobotDetection {
             *context.object_threshold,
         );
 
-        let filtered_detections =
-            filter_by_pixel_position(grid_boxes, *context.lowest_bottom_pixel_position);
+        let mut filtered_detections = grid_boxes;
 
-        let filtered_detections = filter_by_size(
-            filtered_detections,
-            context.camera_matrix,
-            context.robot_to_ground,
-            context.allowed_projected_robot_height,
-        );
+        if *context.enable_filter_by_pixel_position {
+            filtered_detections = filter_by_pixel_position(
+                filtered_detections,
+                *context.lowest_bottom_pixel_position,
+            );
+        }
+
+        if *context.enable_filtered_detections {
```

Comment:
should be called `enable_filter_by_size`

Reply:
Done!